src/sentry/static/sentry/app/views/releases/releaseCommits.jsx

```diff
@@ -161,8 +170,75 @@ const ReleaseCommits = React.createClass({
         </ul>
       </div>
     );
+  },
+
+  render() {
+    if (this.state.loading) return <LoadingIndicator />;
+
+    if (this.state.error) return <LoadingError />;
+
+    let {commitList, activeRepo} = this.state;
+
+    if (!commitList.length) return <this.emptyState />;
+    let commitsByRepository = this.getCommitsByRepository();
+    return (
+      <div>
+        <div className="heading">
+          {Object.keys(commitsByRepository).length > 1
+            ? <div className="row">
+                <div className="commits-header col-xs-1">
+                  <h5>Commits</h5>
+                </div>
+                <div className="commits-dropdown col-xs-11 align-left">
+                  <div className="commits-dropdown col-xs-2">
+                    <DropdownLink
+                      caret={false}
+                      title={
+                        <span
+                          className="icon-arrow-down dropdown"
+                          style={{marginLeft: 3, marginRight: -3}}
+                        />
+                      }
+                    >
```

Comment:
So, this is the one setting both me and @MaxBittker debated changing - putting the final `>` not on its own line.

I think based on these comments, we're going to put it in.